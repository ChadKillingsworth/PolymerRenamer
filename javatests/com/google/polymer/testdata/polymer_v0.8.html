<!doctype html>
<html>
<head>
<title>
</title>
</head>
<body>
<!-- No renames in the next script tag since it's not in dom-module. -->
<script>
  var foo = my.coolSymbolName;
  foo();
</script>
<dom-module name="x-test">
  <template>
    <style>
      symbolA {
      }
    </style>
    <div id="{{foo.dontRenameMe}}" name="{{foo.coolSymbolName}}" selected="0" foo?="{{polymerdoesthis}}">
      {{symbolA}} [[symbolA]] [[foo.dontRenameMe]]
    </div>
    <div selected>
      Another selected div.
    </div>
    <template if="{{symbolA == 42}}">
      symbolA is 42.
    </template>
    Annotated Computed Property: <span>{{symbolA(prop1, coolSymbolName)}}</span>
    Annotated Computed Property 2: <span>{{dontRenameMe(foo.coolSymbolName, symbolA)}}</span>
  </template>
  <script>
    Polymer({
      is: 'x-test'
    });
  </script>
</dom-module>
<x-test cool-symbol-name="value" not-renamed="baz" symbol-b="20"></x-test>
</body>
</html>
