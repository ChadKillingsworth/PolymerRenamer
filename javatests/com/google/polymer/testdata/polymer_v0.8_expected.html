<!DOCTYPE html><html><head>
<title>
</title>
</head>
<body>
<!-- No renames in the next script tag since it's not in dom-module. -->
<script>
  var foo = my.coolSymbolName;
  foo();
</script>
<dom-module name="x-test">
  <template>
    <style>
      symbolA {
      }
    </style>
    <div id="{{foo.dontRenameMe}}" name="{{foo.b}}" selected="0" foo?="{{polymerdoesthis}}">
      {{ab}} [[ab]] [[foo.dontRenameMe]]
    </div>
    <div selected>
      Another selected div.
    </div>
    <template if="{{ab==42}}">
      symbolA is 42.
    </template>
    Annotated Computed Property: <span>{{ab(prop1,b)}}</span>
    Annotated Computed Property 2: <span>{{dontRenameMe(foo.b,ab)}}</span>
    Two-way native element binding: <input value="{{b::input}}">
    Two-way native element binding 2: <input value="{{notRenamed::input}}">
    <x-test b="value" not-renamed="baz" on-symbol-b="b" a-c="20"></x-test>&nbsp;
  </template>
  <script>Polymer({is:"x-test"});</script>
</dom-module>
<x-test b="value" not-renamed="baz" on-symbol-b="coolSymbolName" a-c="20"></x-test>

</body></html>
